### ISSUE TITLE
Characters in strings getting replaced with unicode escape

### ISSUE THREAD

**Issue by meain** [TomWright/dasel#163]
**Describe the bug**
`<` char in json strings is getting replaced with `\u003c`. Similar thing happens to `>`.

**To Reproduce**
1. Create file with this content:
```
{
  "name": "John Doe <email@provider.com>"
}
```
2. `dasel -f filename.json`
```
{
  "name": "John Doe \u003cemail@provider.com\u003e"
}
```

**Expected behavior**
No characters changed.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 1.20.0

**Comment by ypicard** [comment]
I confirm this issue.

**Comment by TomWright** [comment]
This is on my list of things to tackle 👍

**Comment by TomWright** [comment]
This issue is caused when json encoding the content with golangs default JSON marshaler: https://play.golang.org/p/jiQML5SurFF

(nothing to do here, just making a note to myself)

**Comment by TomWright** [comment]
This has been solved in `v1.21.0` with the use of `--escape-html=false`.

```bash
$ dasel -v
dasel version v1.21.0

$ echo '{
  "name": "John Doe <email@provider.com>"
}' | dasel -r json .
{
  "name": "John Doe \u003cemail@provider.com\u003e"
}

$ echo '{
  "name": "John Doe <email@provider.com>"
}' | dasel -r json --escape-html=false .
{
  "name": "John Doe <email@provider.com>"
}
```

This new flag defaults to true by default to maintain backwards compatibility.

**Comment by meain** [comment]
I personally think `--escape-html` should be false by default, but thanks for the fix.

**Comment by TomWright** [comment]
I did consider that but it's a risky change to push out because it would change the default functionality when working with JSON - I don't want to break any existing usages in pipelines and the like.

**Comment by ypicard** [comment]
True - but i don't actually think people actually _expect_ this default behaviour. For sure, many will end up here.

Could we expect this in a major upgrade with breaking changes to the current version?

**Comment by TomWright** [comment]
@ypicard @meain 
I've been doing some reading and since any JSON parser can decode either option this can be considered a purely cosmetic change. As a result I've just pushed a change to set the default to `false` and this will be available in `v1.21.1`.
